Formula/seexpr.rb

```diff
@@ -22,47 +22,59 @@ class Seexpr < Formula
   depends_on "doxygen" => :build
   depends_on "libpng"
 
+  uses_from_macos "flex" => :build
+
+  on_linux do
+    depends_on "mesa"
+    depends_on "mesa-glu"
+  end
+
   def install
-    mkdir "build" do
-      system "cmake", "..", *std_cmake_args, "-DUSE_PYTHON=FALSE"
-      system "make", "doc"
-      system "make", "install"
-    end
+    args = %W[
+      -DCMAKE_INSTALL_RPATH=#{rpath}
+      -DUSE_PYTHON=FALSE
+      -DENABLE_LLVM_BACKEND=FALSE
+      -DENABLE_QT5=FALSE
+    ]
+
+    system "cmake", "-S", ".", "-B", "build", *args, *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
```

Comment:
```suggestion
    system "cmake", "--build", "build"
    system "cmake", "--build", "build", "--target", "doc"
    system "cmake", "--install", "build"
```
Since we run `make doc`. The earlier `test` was a typo.

Reply:
Ah, I'll revise.  That should presumably work.

Reply:
Now everything is working.  We should be all set after Intel macOS is done.